src/CMakeLists.txt

```diff
@@ -87,6 +86,23 @@ endif()
 
 if(LIB_DART_DEBUG) 
     target_link_libraries(dart_dll debug ${LIB_DART_DEBUG})
+else()
+    target_link_libraries(dart_dll debug ${LIB_DART_RELEASE})
```

Comment:
Is the `debug` here as the second parameter correct?

Reply:
Yes if you build only Release from dart as dll but this lib is build als debug. you get an Link error.
So you only build hier release in Pipeline you muss have this. So I Think.